### DISCUSSION TITLE
Local deployment front-end and docker back-end connection

### DISCUSSION THREAD

**Discussion by Flora59** [langgenius/dify#7032] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

The version used is 0.6.15. It is configured through docker and can access the web page through the browser. Now the front-end interface needs to be redeveloped. The front-end files copied from docker are inconsistent with the source code, so it is invalid to mount directly through volumes. Then the front-end code is deployed locally, but the interface console prompts that the cross-origin request has been intercepted: the same-origin policy prohibits reading remote resources.

**Comment by crazywoola** [comment]
Please read our documentations.

**Reply by Flora59** [reply]
which one?I followed the link https://docs.dify.ai/getting-started/install-self-hosted/local-source-code to deploy the frontend page,but how can I connect it to the back in the docker? I tried to revise “api ports” in docker-compose.yaml,and it can successfully open the web page,but I wonder if this can really connet to the back.

BTW,When I modify the font on the front end, it can be updated locally in real time. However, when I modify the rendering of the component in web/app/(commonLayout)/app/(appDetailLayout)/[appid]/workflow, I don't see any real-time update. Is there any file I can refer to for this code part?
